### PR TITLE
Safari Web Push

### PR DESCRIPTION
**Pull request by davidbielik**

### CODE REVIEW

**Comment by davidbielik** [comment]
hey @josh-mccrowell-braze there's some image styling/formatting issues i can't figure out, but otherwise this is ready to go!

File: _docs/_developer_guide/platform_integration_guides/web/push_notifications/safari_mobile_push.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+First, please read and follow our standard [web push integration guide][web-push-integration]. The following steps are only required to support web push on Safari for iOS and iPadOS support.
+
+### Step 1. Create a Manifest File {#manifest}
+
+A [Web Application Manifest][manifest-file] is a JSON file that controls how your website is presented when installed to a user's home screen.
+
+For example, you can set the background theme color and icon that the [App Switcher][app-switcher] uses, whether it renders as full screen to resemble a native app, or whether the app should open in landscape or portrait mode.
+
+Create a new `manifest.json` file in your website's root directory, with the following mandatory fields. The full list of supported fields can be found [here][https://developer.mozilla.org/en-US/docs/Web/Manifest].
```
Comment (froodian) [inline _docs/_developer_guide/platform_integration_guides/web/push_notifications/safari_mobile_push.md:28]:
```suggestion
Create a new `manifest.json` file in your website's root directory, with the following mandatory fields. The full list of supported fields can be found [here](https://developer.mozilla.org/en-US/docs/Web/Manifest).
```

File: _docs/_developer_guide/platform_integration_guides/web/push_notifications/safari_mobile_push.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  "icons": [{
+    "src": "favicon.ico",
+    "sizes": "128x128",
+  }]
+}
+```
+
+### Step 2. Link the Manifest File {#manifest-link}
+
+Add the following `<link>` tag to your website's HTML pointing to where your manifest file is hosted.
```
Comment (froodian) [inline _docs/_developer_guide/platform_integration_guides/web/push_notifications/safari_mobile_push.md:44]: might want to specifically say 

```suggestion
Add the following `<link>` tag to your website's `<head>` element pointing to where your manifest file is hosted.
```

so it doesn't end up buried in some local push-specific html that isn't loaded on the homepage

File: _docs/_developer_guide/platform_integration_guides/web/push_notifications/safari_mobile_push.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+```typescript
+import * as braze from "@braze/web-sdk";
+
+button.onclick = function(){
+    braze.requestPushPermission(() => {
+        console.log(`User accepted push prompt`);
+    }, (temporaryDeny) => {
+        console.log(`User temporarily dismissed push prompt`);
+    });
+};
```
Comment (froodian) [inline _docs/_developer_guide/platform_integration_guides/web/push_notifications/safari_mobile_push.md:77]: can this be 

```suggestion
import { requestPushPermission } from "@braze/web-sdk";

button.onclick = function(){
    requestPushPermission(() => {
        console.log(`User accepted push prompt`);
    }, (temporary) => {
        console.log(`User ${temporary ? "temporarily dismissed" : "permanently denied"} push prompt`);
    });
};
```

?

**Review by froodian — APPROVED** [review]

File: _docs/_developer_guide/platform_integration_guides/web/push_notifications/safari_mobile_push.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+### Step 3. Service Worker {#service-worker}
+
+Your website must have a service worker file that imports the Braze service-worker library, as described in our [web push integration guide][service-worker].
+
+### Step 4. Add To Homescreen {#add-to-homescreen}
+
+![An iphone showing options to bookmark a website and save to the homescreen][add-to-homescreen-img]{: style="max-width:20%;float:right;"}
+
+Unlike major browsers like Chrome and Firefox, you are not allowed to request push permission on Safari iOS/iPadOS unless your website has been added to the user's homescreen.
```
Comment (jocelynzz) [inline _docs/_developer_guide/platform_integration_guides/web/push_notifications/safari_mobile_push.md:60]: To confirm, Chrome or Firefox already allows users to request push permission on iOS/iPadOS?

Comment (davidbielik) [reply _docs/_developer_guide/platform_integration_guides/web/push_notifications/safari_mobile_push.md:60]: yes

**Pull request merged**